### PR TITLE
LCH-7174: Fixing deprecation of module_handler service.

### PR DESCRIPTION
**Pull request by narendradesai**
**Motivation**

Fixes #NNN

**Proposed changes**

**Alternatives considered**

**Testing steps**

### CODE REVIEW

File: src/Command/PqCommands/ContentHubPqCodeCheck.php [hunk 1]
```diff
@@ -79,10 +79,23 @@ protected function configure() {
...
+            if (!empty($module)) {
+              $hookImplementation[$hook][] = $module;
+            }
+          }
+        );
+      }
+      else {
+        if (!empty($moduleList = $moduleHandler->getImplementations($hook))) {
+          $hookImplementation[$hook] = $moduleList;
+        }
```
Comment (kirtigarg2584) [inline src/Command/PqCommands/ContentHubPqCodeCheck.php:113]: Can we try replacing if logic with following 
`$hookImplementation[$hook] = $moduleHandler->getImplementations($hook);`

Comment (narendradesai) [reply src/Command/PqCommands/ContentHubPqCodeCheck.php:113]: I have updated the code with simplified logic.

File: src/Command/PqCommands/ContentHubPqCodeCheck.php [hunk 2]
```diff
@@ -81,11 +82,38 @@ public function getHookImplementation(): array {
     $hookImplementation = [];
     $moduleHandler = $this->drupalServiceFactory->getDrupalService('module_handler');
     foreach (ContentHubAudit::V1_MODULE_HOOKS as $hook) {
-      if (!empty($moduleList = $moduleHandler->getImplementations($hook))) {
+      $moduleList = method_exists($moduleHandler, 'invokeAllWith')
```
Comment (saxumVermes) [inline src/Command/PqCommands/ContentHubPqCodeCheck.php:85]: didn't you mean `'invokeAllWithHook'` for the `method_exists` check?

Comment (saxumVermes) [reply src/Command/PqCommands/ContentHubPqCodeCheck.php:85]: ah, my mistake, all is good!

**Review by saxumVermes — APPROVED** [review]

**Pull request merged**